vllm/config.py

```diff
@@ -56,8 +59,7 @@
 
     ConfigType = type[DataclassInstance]
 else:
-    QuantizationConfig = None
-    ConfigType = type
+    ConfigType = BaseModelLoader = PretrainedConfig = QuantizationConfig = ExecutorBase = PlacementGroup = Any
```

Comment:
This is for arg_utils.py when we evaluate the config class type

Reply:
Will this change break calls to this?

https://github.com/vllm-project/vllm/blob/77073c77bc2006eb80ea6d5128f076f5e6c6f54f/vllm/engine/arg_utils.py#L132-L134

If they're all `Any`, then they'll all appear as built-in?

Reply:
Yes i believe so. idk if we uses PretrainedConfig for docs?